src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,79 @@
+.menu {
+  background-color: #c2c2c2;
+
```

Comment:
1. when the menu is open body- and tab-scrolling shouldn't work;
2. check-list: # 17. Fix menu for small screens (if there is not enough space for all the menu items)


![scroll](https://user-images.githubusercontent.com/89162624/209441754-fbe0bec3-017f-4a46-9e15-9ed756807fed.png)
![scroll-y](https://user-images.githubusercontent.com/89162624/209441755-04ccca32-ea12-43c3-9a14-36674ca3f92c.png)